Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -967,4 +980,455 @@ open class NavigationMapView: UIView {
         
         return candidates
     }
+
+    public var showIntersectionAnnotations: Bool = false {
+        didSet {
+            guard oldValue != showIntersectionAnnotations else { return }
+            intersectionsToAnnotate = nil
+            if showIntersectionAnnotations {
+                NotificationCenter.default.addObserver(self,
+                                                       selector: #selector(didUpdateElectronicHorizonPosition),
+                                                       name: .electronicHorizonDidUpdatePosition,
+                                                       object: nil)
```

Comment:
As of #3475, it’s no longer enough to simply observe electronic horizon updates; the UI needs to actively start and stop updates. This would be the job of something above NavigationMapView, such as NavigationViewController.

Reply:
Added calls to start and stop electronic horizon updates when rebasing.